### ISSUE TITLE
Dark Funnel Campaign 

### ISSUE THREAD

**Issue by LucyeoH-zz** [mozilla/CampaignsforMDM#66]
## Project title: 
The objective of this campaign will be to mobilize our large, international community of volunteers to: 
Find the top sites promoting non-authorized versions of Firefox. 
Identify malicious behaviour 
As a result of this campaign we’ll be able to outreach to sites to either improve the experience or shut them down.

**Comment by couci** [comment]
Lucy wrote the project charter, meeting with the team on the 21st

**Comment by yomanpatil** [comment]
Just had an idea for the website- Can we have a universal dashboard which will show the total number of websites reported by the volunteers all around the world.

Example: [counter on website](https://i2.wp.com/rainastudio.com/wp-content/uploads/2017/10/Add-Visitor-Counter-to-Website.jpg)
 
This will be something like in 2 days volunteers have reported 2000 websites. And we can push the volunteers like 'Hello Mozillians lets try to cross the 5000 websites by this weekend' . We used this  technique in the AADHAR campaign, where we used to push the volunteers like we need 1000  more signup by [campaign end date] and volunteer used to get motivated and work upon that number.
